ramalama/model_factory.py

```diff
@@ -39,6 +41,25 @@ def __init__(
             dm_args = copy.deepcopy(args)
             dm_args.model_draft = None
             self.draft_model = ModelFactory(args.model_draft, dm_args, ignore_stderr=True).create()
+        if (not no_children) and is_split_file_model(model):
+            sm_args = copy.deepcopy(args)
+            sm_args.model_draft = None
+            if is_split_file_model(model):
+                match = re.match(SPLIT_MODEL_RE, model)
+                path_part = match[1]
+                filename_base = match[2]
+                total_parts = int(match[3])
```

Comment:
so IIUC I would pull the model file for the last split, and the files from 0 to <last split-1> will be added as children?